src/ts/components/subtitleoverlay.ts

```diff
@@ -453,8 +466,8 @@ export class SubtitleRegionContainerManager {
    * If the subtitle has positioning information it is added to the container.
    * @param label The subtitle label to wrap
    */
-  addLabel(label: SubtitleLabel): void {
-    const regionName = label.region || 'default';
+  addLabel(label: SubtitleLabel, overlaySize?: {width: number, height: number}): void {
+    const regionName = label.vtt && label.vtt.region ? label.vtt.region.id : label.region || 'default';
```

Comment:
Would it make sense to add brackets here? How is this `||` interpreted if e.g. `region.id` would be undefined?